src/Engine/Component/CameraComponents/CameraComponent.cpp

```diff
@@ -26,11 +26,15 @@ CameraComponent::CameraComponent(float fov, float near, float far) :
 void CameraComponent::init() {    
     float fovRadians;
     fovRadians = glm::radians(m_fov);
+    fovRadians *= 1.2f;
 
-    farPlaneHeight = 2 * glm::tan(fovRadians / 2) * m_far;
+    float planeScale;
+    planeScale = 1.2f;
```

Comment:
Pls can we fix this

Reply:
It's fixed on my branch that I'll make a pr for sometime today

Reply:
Fixed on my branch as well. Feel free to select which implementation to keep